### PR TITLE
Add "suggest an edit" button and minor fix

### PR DESCRIPTION
**Pull request by jonaprieto**
Due to a Github bug, the Agda block inside the details tag is not rendered correctly on the Github website if the block contains blank lines. Fredrik filed an issue on GitHub, and it is currently being investigated. In the meantime, we delete the blank lines in the Agda code import section. The imports have been sorted as well. 

Also, I added a "Suggest an edit" button on the website to make it easier to suggest edits to the Agda code. The button is located at the top right corner of the menu bar. I also fixed the Agda-UniMath title, which now links to the correct URL. 

Check this version here: https://jonaprieto.github.io/agda-unimath/

### CODE REVIEW

**Review by fredrik-bakke — APPROVED** [review]
Amazing additions!

Did you write a script to sort all the imports, and if so, would it be possible to add this as part of the automatic workflow?

File: theme/js/custom.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const link = document.querySelector('.href-git-edit-button');
+if (link) {
+  // console.log(link);
+  var filename = link.getAttribute('href');
```
Comment (VojtechStep) [inline theme/js/custom.js:4]: Minor nitpick, but why have `var` instead of `let` here?

Comment (jonaprieto) [reply theme/js/custom.js:4]: > Amazing additions!
> 
> Did you write a script to sort all the imports, and if so, would it be possible to add this as part of the automatic workflow?

It's possible, sure. Here it is in the meanwhile,
https://gist.githubusercontent.com/jonaprieto/4fc4a98e147ef60c15f5ae6b03b04205/raw/72ef7f16b7f2ea60f6512145ec26b1b758251a76/sort-imports.py

Comment (jonaprieto) [reply theme/js/custom.js:4]: > Minor nitpick, but why have `var` instead of `let` here?
I'll change it in the next PR ;)

**Comment by jonaprieto** [comment]
> Amazing additions!
> 
> Did you write a script to sort all the imports, and if so, would it be possible to add this as part of the automatic workflow?

It's possible, sure. Here it is in the meanwhile,
https://gist.githubusercontent.com/jonaprieto/4fc4a98e147ef60c15f5ae6b03b04205/raw/72ef7f16b7f2ea60f6512145ec26b1b758251a76/sort-imports.py

**Comment by fredrik-bakke** [comment]
> Due to a Github bug, the Agda block inside the details tag is not rendered correctly on the Github website if the block contains blank lines. Fredrik filed an issue on GitHub, and it is currently being investigated. In the meantime, we delete the blank lines in the Agda code import section. The imports have been sorted as well.

This issue can now be followed here #489.

**Pull request merged**